assignment4.Rmd

```diff
@@ -71,26 +82,36 @@ summary(pca)
 
 #We can look at this to get an idea of which components we should keep and which we should drop
 
-plot(pca, type = "lines")
+#PCA Variance Plot 1: Assistments
+plot(pca, type = "lines", main = "Assistments PCs")
 ```
 
-## Think about which components you would drop and make a decision
+## Q1:Think about which components you would drop and make a decision
+
+## A1: I would drop PC6 since PC1-PC5 capture approx 91% of the variance based on the cumulative proportion shown in the summary.
```

Comment:
Analysis of the correlogram between principle components and mean_correct indicate that PC6 has a relatively high correlation with mean_correct compared to the other components. Thus, dropping PC6 may not be the best decision if we want to maintain info about mean_correct. Similarly PC1-PC5 all show moderate to high correlations with at least one of the original variables so dropping any of them could result in significant info loss.